### ISSUE TITLE
Connects to github but throws errors in logs and on widget

### ISSUE THREAD

**Issue by Haui1112** [nextcloud/integration_github#83]
Hi there,
I'm running NC 27 in Docker behind a local ssl nginx reverse proxy and installed the app today.
- Connection to Github works (github side recognizes the key being used)
- But the frontend shows an error "error connecting to github".
- Meanwhile the logs are spammed with the following error:

> Creation of dynamic property OCA\Github\AppInfo\Application::$config is deprecated at /var/www/html/custom_apps/integration_github/lib/AppInfo/Application.php#42
> and
> Creation of dynamic property OCA\Github\AppInfo\Application::$container is deprecated at /var/www/html/custom_apps/integration_github/lib/AppInfo/Application.php#41

I'm not sure how to proceed. Let me know if you need anything. Thank you for making this project in the first place.
Have a good one!

**Comment by julien-nc** [comment]
Hi, thanks for the bug report. 

Looking at the implementation, I guess you're seeing this error message in the dashboard.
There might be more information in the response of the request getting the notifications. Could you check that?
* Open the browser dev tools (press F12 on most browsers) and go in the network tab
* Reload the dashboard page
* Filter the request list (in the network tab) with "github/notifications"
* You should see one request, select it
* Check the content of the response tab

> Creation of dynamic property OCA\Github\AppInfo\Application::$config is deprecated

It's been fixed already but is not released yet.

**Comment by Haui1112** [comment]
Hi there!

It was user error. Github has both standard tokens and "finegrained tokens". I misclicked and chose the finegrained one. I assume that is why I could make a connection but nothing else.

However, I did change that and now it works like a charm! Thank you for responding, getting that response and revisiting the whole process to search for errors was what made me stumble upon it. :)

Thank you for providing this software and have a wonderful day!
